src/pages/learn.js

```diff
@@ -65,12 +69,19 @@ const HeaderContent = styled('div')`
     margin-right: auto;
 `;
 
-const Article = styled('article')`
+const CardContainer = styled('div')`
     padding: ${size.medium};
 `;
 
 const StyledFilterBar = styled(FilterBar)`
-    padding-bottom: ${size.medium};
+    margin: 0 auto;
+    padding-bottom: ${size.large};
+    width: 95%;
```

Comment:
using % widths aside from say 100 have some unintended effects when it comes to being responsive and with the filter bar at 94 I'm sure we can think about this a bit more. It would be safer for us to use some padding instead, would that get the job done?